src/converse-muc-views.js

```diff
@@ -853,12 +853,15 @@ converse.plugins.add('converse-muc-views', {
             },
 
             parseMessageForCommands (text) {
-                if (_converse.muc_disable_moderator_commands) {
+                if (_converse.muc_disable_moderator_commands && _converse.muc_disable_moderator_commands.constructor !== Array) {
```

Comment:
Please use [_.isArray](https://lodash.com/docs/4.17.11#isArray) here instead and wrap the line.